portal/src/graphql/portal/AuthenticationLoginIDSettings.tsx

```diff
@@ -304,6 +306,7 @@ const AuthenticationLoginIDSettings: React.FC<Props> = function AuthenticationLo
             />
 
             <CheckboxWithContent
+              ariaLabel={renderToString("AuthenticationWidget.excludeKeywords")}
```

Comment:
`ariaLabel: React.ReactNode`

Reply:
`ariaLabel` in `Checkbox` component in fluent UI also only takes string

Reply:
Fair enough.